patches/net/minecraft/core/DefaultedMappedRegistry.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/core/DefaultedMappedRegistry.java
++++ b/net/minecraft/core/DefaultedMappedRegistry.java
+@@ -40,6 +_,12 @@
+         return resourcelocation == null ? this.defaultKey : resourcelocation;
+     }
+ 
++    @Nullable
++    @Override
++    public ResourceLocation getKeyOrNull(T element) {
++        return super.getKey(element);
```

Comment:
Should this be `super.getKey`? Isnt that going to invoke the impl from `MappedRegistry` ignoring the `defaultValue` in `DefaultedMappedRegistry`?

Reply:
That is the entire point of the method? To ignore the `defaultValue`

Reply:
Oh.. ooh, I completely misunderstood the PR then, during quick skim through my notifications.

Just felt off since that's also the default generated body for overrides and methods like this that just invoke super never look right to me, always gotta double check them.